README.md

```diff
@@ -78,6 +78,7 @@ In addition, many plugins can be found on the [npm registry](https://www.npmjs.c
   [omi-chart](https://github.com/Tencent/omi/tree/master/components/chart) | Omi | ✔ | ✔
   [react-chartjs-2](https://github.com/jerairrest/react-chartjs-2) | React | ✔ | ✔
   [vue-chartjs](https://github.com/apertureless/vue-chartjs/) | Vue.js | ✔ |
+  [vue-chart-3](https://github.com/victorgarciaesgi/vue-chart-3) | Vue.js  | ✔ | ✔
```

Comment:
According to your [package.json](https://github.com/victorgarciaesgi/vue-chart-3/blob/7f1048bbe7e457cd8d4f938eba6c50abf3659e56/package.json#L45) it doesnt work with chart.js v2 since you require chart.js version 3.1.0 or higher.

Also in your readme you say its a rewrite for chart.js v3, no specification of chart.js v2

So the checkmark for v2 should be removed or your wrapper needs to be updated.

Reply:
Thanks for pointing out. i removed the checkmark.